Dockerfile.proto

```diff
@@ -0,0 +1,70 @@
+# syntax=docker/dockerfile:1
+
+ARG UBUNTU_VERSION=22.04
+ARG TFSERVING_VERSION=2.9.0
+
+# Build image, genearates proto files
+FROM arm64v8/ubuntu:${UBUNTU_VERSION}  AS build-image
+
+ARG TFSERVING_VERSION
+
+RUN <<EOF
+apt-get update && apt-get install -y --no-install-recommends \
+python3 \
+python3-pip \
+git \
+&& rm -rf /var/lib/apt/lists/*
+EOF
+
+# grpcio-tools contains protoc, which allows to build the pb2.py files in the install-tf.sh script.
+# Installing directly grpcio-tools result in an older version of protobuf.
+RUN pip install  --no-cache-dir \
+      grpcio==1.46.3 \
```

Comment:
lift out this version as an build-arg GRPC_VERSION - the same should probably go for the other packages

Reply:
Done